pkgs/development/interpreters/python/cpython/default.nix

```diff
@@ -88,6 +88,33 @@ let
     "$out/bin/python"
   else pythonForBuild.interpreter;
 
+  sysconfigdataHook = with stdenv.hostPlatform; let
```

Comment:
I'd appreciate a comment here explaining briefly what sysconfig is, what this hook does and why.